test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java

```diff
@@ -150,6 +166,46 @@ public void testWorkbenchProjectPersistence() {
         assertThat(containersResponse.getResult().getContainers().get(0).getContainerId()).isEqualTo(CONTAINER_ID);
     }
 
+    @Test
+    public void testGitHooksPersistence() {
+        // get git hook dir location
+        String gitHooksRemoteDir = "/opt/eap/standalone/data/bpmsuite/git/hooks"; //TODO for 7.1 bpmsuite -> kie
+
+        updateWorkbenchDeployment(gitHooksRemoteDir);
```

Comment:
Will git hooks environment variable be part of template parameters?

Reply:
Not decided yet